modules/account-lib/src/coin/hbar/keyPair.ts

```diff
@@ -34,6 +35,22 @@ export class KeyPair extends Ed25519KeyPair {
     return result;
   }
 
+  /**
+   * Hedera keypair in the SDK supported format
+   *
+   * @returns { SDKKeyPair } the keys in SDK format
+   */
+  getSDKKeys(): SDKKeyPair {

```

Comment:
This and `getKeys` are the same method. I would remove it and make any conversion needed where it is called.

Reply:
Also, why do we need the `raw` option in `getKeys`?

Reply:
Sure, that approach sounds good, so we decouple from the raw implementation.